portal/server/routes/contract/bond/index.js

```diff
@@ -308,15 +308,15 @@ router.post('/contract/:_id/bond/:bondId/issue-facility', async (req, res) => {
   ];
 
   /**
-   * Add `coverDateConfirmed: true` property to the bond.
+   * Add coverDateConfirmed: true property to the bond.
```

Comment:
please can we revert this line change

Reply:
Done

Reply:
might be an issue with my git but im not seeing it

Reply:
This is done. I cannot see at my end.